lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -88,7 +88,18 @@
       - The database state
     required: false
     default: present
-    choices: [ "present", "absent" ]
+    choices: [ "present", "absent", "dump", "restore" ]
+    backup:
+      - Back up to a destination, requires target as well. Supported formats are
+        .gz, .tar, .bz, .xz, and .sql
+  restore:
+    version_added: "2.2"
+    description:
+      - Restore from a destination. See backup for supported options
```

Comment:
I'm inclined to think these were originally intended as part of the description. I'll migrate.